style.css

```diff
@@ -1,29 +1,31 @@
-.hljs.line-numbers,
-.hljs.selected-lines {
+.hljs.shcb-code-table {
 	display: table;
 	width: 100%;
 }
 
-.hljs.line-numbers .loc,
-.hljs.selected-lines .loc {
+.hljs.shcb-code-table > .shcb-loc {
 	display: table-row;
 	width: 100%;
 }
 
-.hljs.line-numbers {
+.hljs.shcb-line-numbers {
 	border-spacing: 0;
 	counter-reset: line;
 }
 
-.hljs.line-numbers .loc {
+.hljs.shcb-line-numbers > .shcb-loc {
 	counter-increment: line;
 }
 
-.hljs.line-numbers .loc > span {
+.hljs.shcb-code-table > .shcb-loc > span::after {
+	content: "\200B"; /* Hack to force empty spans to render as table cells */
+}
```

Comment:
This is the downside to use the CSS hack for rendering new lines, you lose the new line when you copy/paste. Since pseudo-elements are by definition not "selectable".

![2020-04-22 16-20-09 2020-04-22 16_21_03](https://user-images.githubusercontent.com/1246453/80042994-52200a80-84b5-11ea-9f70-c35368a463ca.gif)

Reply:
Good find. What about 297d469?

Reply:
I like this much better! 👍